addons/account/models/account_move.py

```diff
@@ -3388,8 +3397,11 @@ def _copy_data_extend_business_fields(self, values):
     @api.multi
     def copy_data(self, default=None):
         res = super(AccountMoveLine, self).copy_data(default=default)
-        if self._context.get('include_business_fields'):
-            for line, values in zip(self, res):
+        for line, values in zip(self, res):
```

Comment:
this fix is not related to the title of the commit it belongs to; make an additionnal commit or squash everything